### ISSUE TITLE
Empty save-file is created when id is empty

### ISSUE THREAD

**Issue by rimms** [jubatus/jubatus#1038]
If ``save`` is called with empty ``id``, ``RPCError`` will be raised.
And then, an empty file is created.

```py
client.save('')
RPCError: empty id is not allowed
```
```
$ ls /tmp
/tmp/172.16.100.1_9199_classifier_.jubatus
$ du /tmp/172.16.100.1_9199_classifier_.jubatus
0       /tmp/172.16.100.1_9199_classifier_.jubatus
```

Log messages:
```
2015-08-22 18:51:45,830 31546 INFO  [server_base.cpp:130] starting save to /tmp/172.16.100.1_19295_classifier_.jubatus
2015-08-22 18:51:45,830 31546 WARN  [rpc_server.cpp:46] exception in RPC thread: Dynamic exception type: jubatus::core::common::exception::runtime_error::what: empty id is not allowed
    #0 [jubatus::core::common::exception::error_at_file_*] = ../jubatus/server/framework/save_load.cpp
    #0 [jubatus::core::common::exception::error_at_line_*] = 105
    #0 [jubatus::core::common::exception::error_at_func_*] = void jubatus::server::framework::save_server(FILE*, const jubatus::server::framework::server_base&, const string&)
```

**Comment by rimms** [comment]
:+1: LGTM

**Comment by rimms** [comment]
Fixed via #1039
